solana/indexer/solana_indexer.go

```diff
@@ -103,23 +94,10 @@ func New(config config.Config) *SolanaIndexer {
 }
 
 func (s *SolanaIndexer) Start(ctx context.Context) error {
-	ticker := time.NewTicker(s.config.SolanaIndexerRetryInterval)
-	defer ticker.Stop()
-
-	go func() {
-		for {
-			select {
-			case <-ctx.Done():
-				s.logger.Info("context cancelled, stopping retry ticker")
-				return
-			case <-ticker.C:
-				err := s.RetryUnprocessedTransactions(ctx)
-				if err != nil {
-					s.logger.Error("failed to retry unprocessed transactions", zap.Error(err))
-				}
-			}
-		}
-	}()
+	go s.ScheduleRetries(ctx, s.config.SolanaIndexerRetryInterval)
+
+	go jobs.NewCoinStatsJob(s.config, s.pool).
+		ScheduleEvery(ctx, 5*time.Minute)
```

Comment:
Config for this interval? TBH not sure if there's a benefit. Deploying a new config is much like deploying a code change 🤷

Reply:
In late to say at the very least these should be named constants defined in a centralized config object instead of inline in a function. But that's a nitpick.

Reply:
why?

Reply:
I think if the initialization of jobs are all short one liners and listed together that's probably good enough. But if there initialized all over the place or are more complicated to init it may get harder to grok